packages/manager/modules/pci/src/projects/project/kubernetes/components/autoscaling/autoscaling.html

```diff
@@ -0,0 +1,110 @@
+            target="_blank"
+            rel="noopener"
+        >
+            <span
+                data-translate="kubernetes_node_pool_autoscaling_description_link"
+            ></span>
+            <span
+                class="oui-icon oui-icon-external_link"
+                aria-hidden="true"
+            ></span>
+        </a>
+    </div>
+
+    <!--Autoscaling-->
+    <div class="mb-4">
+        <oui-field
+            data-label="{{:: 'kubernetes_node_pool_autoscaling_autoscale' | translate}}"
+            data-size="xl"
+        >
+            <oui-switch
+                name="autoscaling-autoscale-value"
+                model="$ctrl.model.autoscale"
+                on-change="$ctrl.onAutoscaleChanged({status: modelValue})"
+            >
+            </oui-switch>
+            <strong
+                data-translate="{{'kubernetes_node_pool_autoscaling_autoscale_toggle_' + $ctrl.model.autoscale }}"
+            ></strong>
+        </oui-field>
+    </div>
+</div>
+
+<!--Autoscaling steeper body-->
+<div data-ng-if="$ctrl.displayNodePoolSizing">
+    <!--Autoscaling lowest config-->
+    <div class="mb-4" data-ng-show="$ctrl.model.autoscale">
+        <oui-field
+            data-label="{{ ::'kubernetes_node_pool_autoscaling_lowest_nodes_size' | translate }}"
+            data-size="xl"
+        >
+            <oui-numeric
+                disabled="!$ctrl.model.autoscale"
+                id="autoscaling-node-pool-lowest-value"
+                name="autoscaling-node-pool-lowest-value"
+                min="$ctrl.model.nodes.lowest.min"
+                model="$ctrl.model.nodes.lowest.value"
+                max="$ctrl.model.nodes.lowest.max"
+                on-change="$ctrl.onLowestValueChange({value: modelValue})"
+                required
+            >
+            </oui-numeric>
+        </oui-field>
+    </div>
+
+    <!--Autoscaling desired config-->
+    <div class="mb-4" data-ng-show="!$ctrl.model.autoscale">
+        <oui-field
+            data-label="{{ ::'kubernetes_node_pool_autoscaling_desired_nodes_size' | translate }}"
+            data-size="xl"
+        >
+            <oui-numeric
+                id="autoscaling-node-pool-desired-value"
+                name="autoscaling-node-pool-desired-value"
+                min="$ctrl.model.nodes.desired.min"
+                model="$ctrl.model.nodes.desired.value"
+                max="$ctrl.model.nodes.desired.max"
+                on-change="$ctrl.onDesiredValueChange({value: modelValue})"
+                required
+            >
+            </oui-numeric>
+        </oui-field>
+
+        <oui-message
+            data-type="warning"
+            data-ng-if="$ctrl.model.nodes.desired.value < $ctrl.LOWEST_VALUE"
+        >
+            <p data-translate="kubernetes_node_pool_autoscaling_desired_nodes_warning"></p>
+        </oui-message>
+    </div>
+
+    <!--Autoscaling highest config-->
+    <div class="mb-4" data-ng-show="$ctrl.model.autoscale">
```

Comment:
```suggestion
    <div class="mb-4" data-ng-if="$ctrl.model.autoscale">
```

Reply:
Thanks for this suggestion.
Initially i used `ng-if` directive but the UI produce an uncomfortable display (take time to build the full elements).
To have a smoothly display i used  `data-ng-show` to avoid to rebuild elements.